### ISSUE TITLE
Some keys no longer accessible in ./Bazecor-1.0.0-beta13.AppImage

### ISSUE THREAD

**Issue by erwin** [Dygmalab/Bazecor#309]
**Describe the bug**
* Apps Calc Key
* Camera Key

# Apps Calc - Not Selectable on bottom keyboard

![image](https://user-images.githubusercontent.com/11534217/148175139-ee263b10-acfa-4a5d-8b9e-85555d6c785c.png)

# Camera Key - Shows up as ".Camera" under "Selected Key" but can't be changed.

![image](https://user-images.githubusercontent.com/11534217/148175262-1ba502ce-7bf6-4c00-b5a1-ce63ccff1d95.png)

The New UI is great but a lot of the keys you already had support for are now dropped.

https://github.com/Dygmalab/Bazecor/pull/232

https://github.com/Dygmalab/Bazecor/issues/90

---

UPDATE:

Same goes for these keys:

* `Display Bright +`
* `Display Bright -` 

Basically it seems that any key not explicitly listed on the bottom keymap isn't able to be added, even though Bazecor even has a record for most of them... For example, the key name does still show up on the keyboard. No way to assign that value to a different key now though.

Right now there is a `F13+` button that allows selecting the `F13 - F24` keys, but there is no similar button to select the unlisted media keys or `XF86*` keys...

**Comment by AlexDygma** [comment]
@erwin !! Check the new Beta 14 to see this issue solved 😄

**Comment by erwin** [comment]
Yes, by checking out the `newui` branch, I do see the new key selection UI. Thanks!

Colors / effects look much better... Feels very modern!

Instead of, or perhaps in addition to, the UI for selecting common keys, I wish you had a way to just specify a key code from with a two column combo box of names & codes.

An editable keyname / keycode combo box would both more compact and more flexible than two whole rows in the UI that still only cover a very small fraction of all the possible "media keys":

![image](https://user-images.githubusercontent.com/11534217/164952192-825277dd-cd06-40d5-a4c6-0850562b4695.png)

Here's a multi-column comboxbox dropdown (Run it, see 3rd row: `table-style`)... Something like this except letting the user override both the name and the keycode

  https://www.grapecity.com/wijmo/demos/Input/ComboBox/Multi-column/purejs

Also, the new design for the keyboard requires a ton of screen real-estate though, and does not scale nicely as the screen the windows is expanded/contracted...

# UI doesn't really scale: Small

![image](https://user-images.githubusercontent.com/11534217/164952131-79af45b3-60b9-4af1-80e3-a7591f2c5c8f.png)


# UI doesn't really scale: Large

![image](https://user-images.githubusercontent.com/11534217/164952149-b549196d-3231-4d25-8cc2-992f92b9d4ac.png)

# Minor Error: off by one on Macro keys

When using the new "Macro" buttons, there seems to be an off by one error... Selecting "Macro 1" on the keyboard shows "Macro 2" on the top layout...

Select: 
![image](https://user-images.githubusercontent.com/11534217/164952012-7d54e3aa-c820-4232-9d14-82dded8ec99c.png)

Shows:
![image](https://user-images.githubusercontent.com/11534217/164952020-ce36f3e9-1250-4085-8dfa-e19fc46a3f3b.png)
